src/frontend/src/constants.js

```diff
@@ -66,9 +66,9 @@ export const DEFAULT_CRIME_IMPORTANCE = 1;
 // URLS
 export const MAPBOX_GEOCODING_URL = "https://api.mapbox.com/geocoding/v5/mapbox.places";
 export const REALTOR_BASE_URL =
```

Comment:
I think it would be worth copying your comment "A breakdown of how these endpoints work for future reference" into the code here so that anyone who encounters these endpoints in the future will have a roadmap for what to look at without having to track down this PR.